### ISSUE TITLE
[Enhancement] Active hotbeds reworks

### ISSUE THREAD

**Issue by SanderDevisscher** [inbo/alien-species-portal#103]
# Feature title
The title of the feature requiring improvement or description detailing which feature is concerned.
Please use the following title template if possible: 
- main tab > tab > feature title (example: global indicators > taxa > taxa table)

# Description
last 14 days
reactive pull api waarnemingen 
maximum refresh rate ? 

# (Optional) Needed changes in backoffice

# (Optional) Suggestion how to implement change

# (Optional) Additional less important changes 
nice to haves

**Comment by mvarewyck** [comment]
> Might be naive but I think this can be achieved by doing a daily pull of the waarnemingen.be API. We have the necessary credentials and 3 example scripts to get access.
> 
> the example scripts: [waarnemingenAPI.zip](https://github.com/user-attachments/files/16032791/waarnemingenAPI.zip)
> 
> Alternatively, with some minor changes to these scripts, we can create a GDPR proof version of the data in a googlesheet. Which do you prefer?

@SanderDevisscher Yes, based on the scripts you shared, you can do a daily API pull. I see no need to write the data to a googlesheet, I would just add the data to the S3 bucket with a script from aspbo.

**Comment by SanderDevisscher** [comment]
@mvarewyck best to put it on the PRD bucket directly then ?

**Comment by mvarewyck** [comment]
> @mvarewyck best to put it on the PRD bucket directly then ?

Yes, if you don't want to do a daily PR from UAT to PRD.
Then in the data processing script I would
* add some checks on the data structure (e.g. no. of rows >0, column names) before the data is pushed to the bucket. 
* when the checks fail and there is no update of the data, make sure we get a notification (automatically with github actions?)

**Comment by SanderDevisscher** [comment]
@mvarewyck 0 rows is a real possibility in the winter when they are not active. The app should display a disclaimer when this the case. 

@TheJenne18 can you confirm the PRD - bucket exists and can be writen to by github actions from the aspbo repo ?

**Comment by SanderDevisscher** [comment]
transfer to Aspbo => daily workflow directly to PRD bucket with some workflow tests
